Source/Core/EllipsoidGeometry.js

```diff
@@ -206,6 +206,10 @@ define([
         var cosTheta = new Array(slicePartitions);
         var sinTheta = new Array(slicePartitions);
 
+        if ((radii.x === 0) || (radii.y === 0) || (radii.z === 0)) {
```

Comment:
Move this up to the top.  These kind of checks should be the first thing that happens in the `createGeometry` functions.  No sense doing any calculations if the geometry is going to be undefined.